.changeset/clean-camels-act.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sitemark/exifr": patch
+---
+
+Fixed isBrowser check to work properly in test environments
```

Comment:
Technically this is for compatibility with nodejs 22 which added native fetch, and as such added the global navigator object